### ISSUE TITLE
cactus:v1.1.0 Docker image missing shared libraries for kent binaries

### ISSUE THREAD

**Issue by nathanweeks** [ComparativeGenomicsToolkit/cactus#291]
The cactus v1.1.0 Docker image is missing a couple libraries needed for the kent binaries:
```
$ docker run -it --rm quay.io/comparative-genomics-toolkit/cactus:v1.1.0
...
root@64212e10594b:/data# ldd /usr/local/bin/faToTwoBit  | grep 'not found'
	libkrb5.so.3 => not found
	libk5crypto.so.3 => not found
root@64212e10594b:/data# apt update && apt install -y --no-install-recommends libkrb5-3 libk5crypto3
...
root@64212e10594b:/data# ldd /usr/local/bin/faToTwoBit  | grep 'not found'
root@64212e10594b:/data# 
```

**Comment by glennhickey** [comment]
That didn't take long...   But thanks..  Will begin a patch.

**Comment by glennhickey** [comment]
I'd just assumed they were static. @diekhans do you know if there are static binaries available for these tools?  

@nathanweeks I'm scrolling back through HAL issues as this rings a bell, and it looks like you were ahead of me 
* https://github.com/harvardinformatics/hal/compare/1fe29b80c06161f45dc74f2624878aa8fd0a8629..fffdac7ea8443b7d3823e44687f04c8e0ad8b3a5
* https://github.com/ComparativeGenomicsToolkit/hal/issues/108#issuecomment-632954072

What a pain.  Guess I'll have to take your src approach for the docker -- then try to use that to make static versions for the binary release.

**Comment by glennhickey** [comment]
This should be fixed in 1.1.1.
